docs/src/main/sphinx/release/release-479.md

```diff
@@ -0,0 +1,213 @@
+* Add `queryText` as a regular expression in resource group selector. ({issue}`27129`)
+* Require JDK 25 to build and run Trino. ({issue}`27153`)
+* {{breaking}} The configuration property `task.statistics-cpu-timer-enabled` is now defunct and must be removed. ({issue}`27504`)
+* Deprecate `EXPLAIN (TYPE LOGICAL)`. `EXPLAIN (TYPE DISTRIBUTED)` should be used instead. ({issue}`27434`)
+* Remove `prefer_streaming_operators` session property. `SET SESSION task_concurrency=1` can be used to achieve the same 
+  behaviour as `prefer_streaming_operators`. ({issue}`27506`)
+* Improve performance of remote data exchanges on CPUs supporting the required SIMD extensions. This can be disabled by 
+  setting `exchange.experimental.vectorized-serde.enabled=false`. ({issue}`27426`, {issue}`26919`)
+* Improve performance of {func}`array_sort` function. ({issue}`27272`)
+* Improve performance of {func}`repeat` function. ({issue}`27369`)
+* Improve performance of data exchanges involving variable width data. ({issue}`27377`)
+* Fix query failure when one of the branches of a `UNION` is known to produce zero rows during query planning. ({issue}`21506`)
+* Fix configuring partitioned layout for spooling protocol. ({issue}`27247`)
+* Fix `EXPLAIN (TYPE IO)` failure when query predicate contains a type which cannot be cast to `varchar`. ({issue}`27433`)
+
+## Security
+
+## Web UI
+
+* Render long single-line query string correctly in preview UI query details page. ({issue}`27328`)
+* Render catalog properties correctly in preview UI. ({issue}`27327`)
+
+## JDBC driver
+
+* Add a `extraHeaders` option to support sending arbitrary HTTP headers. ({issue}`15826`)
+
+## Docker image
+
+* Update JDK to 25.0.1. ({issue}`27117`)
+
+## CLI
+
+* Add a `--extra-header` option to support sending arbitrary HTTP headers. ({issue}`15826`)
+* Fix TLS connection failures when using unqualified (single-label, without ".") hostnames. ({issue} `27478`)
+
+## BigQuery connector
+
+* Fix query failure when reusing `query` table function result. ({issue}`27573`)
+
+## Blackhole connector
+
+## Cassandra connector
+
+## ClickHouse connector
+
+* {{breaking}} Replace `join-pushdown.with-expressions` configuration property with `deprecated.join-pushdown.with-expressions`. 
+  The new property will be removed in a future version.
+
+## Delta Lake connector
+
+* Add `APPLICATION_DEFAULT` authentication type for GCS. ({issue}`26984`)
+* {{breaking}} Remove support for unauthenticated access when GCS authentication type is set to `SERVICE_ACCOUNT`. ({issue}`26984`)
+* Rename `s3.exclusive-create` config to `delta.s3.transaction-log-conditional-writes.enabled`. ({issue}`27372`)
+* Fix incorrect results for queries involving `IS NOT DISTINCT FROM`. ({issue}`27213`)
+* Fix failure when writing to tables created by Databricks 17.3. ({issue}`27100`)
+* Fix failure when checking Azure hierarchical namespaces to avoid requiring additional storage permissions. ({issue}`27278`)
```

Comment:
The phrasing sounds weird. It read as if there's a failure that results in requiring additional storage permissions (which makes little sense), and that we're fixing such failure so that the user doesn't have to specify additional storage permissions 🤔. Is the "to" meant to be an "and"?

TL;DR, I have no idea what this is about from reading the item. It's probably simpler to just say "Fix failure when checking Azure hierarchical namespaces".